manifests/init.pp

```diff
@@ -37,6 +37,8 @@
   Boolean $cvmfs_repo_list                                       = true,
   Optional[Integer] $cvmfs_dns_min_ttl                           = undef,
   Optional[Integer] $cvmfs_dns_max_ttl                           = undef,
+  Optional[String] $cvmfs_alien_cache                            = undef,
+  Optional[String] $cvmfs_shared_cache                           = undef
```

Comment:
HI,

CVMFS_SHARED_CACHE is strictly yes, no or undef so an Optional ENUM makes sense I think.

Reply:
For sure!